### PR TITLE
Pranita_Assignment_21_Done_1hour

### PR DESCRIPTION
**Pull request by PranitaPJun20**
image

swap

### CODE REVIEW

File: src/pranitaPJun20/FibonacciSeries.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+			iterationValue++;
+		}
+		/*
+		 * for(int iterationValue=2;iterationValue<=n;iterationValue++) { int
+		 * term3=term1+term2; System.out.print(","+term3); term1=term2; term2=term3; }
+		 */
+	}
+
+	public static void main(String[] args) {
+		int term1 = 1, term2 = 1;
```
Comment (AditiGJUN20) [inline src/pranitaPJun20/FibonacciSeries.java]: Are you using term1, term2 for asking user to start Fibonacci series from given number?
i.e. user wants to start series from 0 or 2 or 3?
If not then you can initialize these variables in core method rather writing in Main().

File: src/pranitaPJun20/FibonacciSeries.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+ */
+package pranitaPJun20;
+
+public class FibonacciSeries {
+//Method to find Fibonacci series for given no of terms
+	static void printFibonacciNumbers(int num1, int num2, int length) {
+		int iterationValue = 2;// iteration starts from 3rd term
+		int term1 = num1;
+		int term2 = num2;
+		System.out.print(term1 + "," + term2);// Printing values at index 0 and 1
```
Comment (AditiGJUN20) [inline src/pranitaPJun20/FibonacciSeries.java]: Without printing hardcoded values you can achieve expected output by making some minor changes in initialization of term1, term2 and iterationValue

File: src/pranitaPJun20/SwapVariables.java [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+	}*/
+
+	public static void main(String[] args) {
+		Scanner scanner = new Scanner(System.in);
+		System.out.println("Enter first number");
+		int num1=scanner.nextInt();
+		System.out.println("Enter second number");
+		int num2=scanner.nextInt();
+		SwapVariables swapVariables = new SwapVariables();
+		swapVariables.swapNumberUsingArithmaticOperator1(num1, num2);
```
Comment (AditiGJUN20) [inline src/pranitaPJun20/SwapVariables.java:36]: If you're using only one method to call it exactly once you can write it as,
**new SwapVariables(). swapNumberUsingArithmaticOperator1(10, 20)**

File: src/pranitaPJun20/SwapVariables.java [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+Program 1: Swap two numbers without using third variable.
+Input: num1= 10, num2 =20
+output: num1=20, num2 =10 
+ */
+package pranitaPJun20;
+
+import java.util.Scanner;
+
+public class SwapVariables {
```
Comment (AditiGJUN20) [inline src/pranitaPJun20/SwapVariables.java:12]: Well Written..

**Review by AditiGJUN20 — CHANGES_REQUESTED** [review]

**Review by AditiGJUN20 — APPROVED** [review]
Changes approved.

File: src/pranitaPJun20/FibonacciSeries.java (hunk 1 above)
Comment (PranitaPJun20) [reply src/pranitaPJun20/FibonacciSeries.java]: Changes done.Initialized variables in core method.

File: src/pranitaPJun20/FibonacciSeries.java (hunk 2 above)
Comment (PranitaPJun20) [reply src/pranitaPJun20/FibonacciSeries.java]: Thank you for guiding me in optimizing the code.Changes done

File: src/pranitaPJun20/SwapVariables.java (hunk 3 above)
Comment (PranitaPJun20) [reply src/pranitaPJun20/SwapVariables.java:36]: Yes Aditi you are correct.From next-time I will write like this.

File: src/pranitaPJun20/SwapVariables.java (hunk 4 above)
Comment (PranitaPJun20) [reply src/pranitaPJun20/SwapVariables.java:12]: Thank you :)

**Pull request merged**